app/src/main/res/values-es-rES/strings.xml

```diff
@@ -102,7 +102,7 @@
     <string name="until">hasta</string>
     <string name="until_end">hasta el final</string>
 
-    <string name="offset_at">at</string>  
```

Comment:
hrm. These at/before/after offset strings seem tricky if not impossible to translate. There isn't any context provided - I think I need to fix that.

My Spanish is pretty rusty, but I think its the difference between displaying "a la" 1 PM vs "a las" 5 PM.